uploading/uploading-router.js

```diff
@@ -41,12 +41,12 @@ router.post("/:user_id/uploading", (req, res) => {
        * Creates an random id based on a very large number
        * @returns {integer}
        */
-      function createId() {
-        function getRandom(max) {
-          return Math.floor(Math.random() * Math.floor(max));
-        }
-        return getRandom(2 ^ 53) ^ 13;
-      }
+      // function createId() {
```

Comment:
If we're not using this we can just delete it. 😄

Reply:
I was getting an error for my initial pass at uploading, that the values provided violated unique constraints. Will clean that out as the database now handles incrementing the id.